docs/development/running-tests/README.md

```diff
@@ -701,15 +701,18 @@ you can access both from inside a VM with nat/bridged networking as follows:
 
 ```shell
 # Start ng serve middleware binding to all interfaces
-PROXY_HOSTNAME=<your local IP address> npm run serve
+FE_HOST=<your local IP address> PROXY_HOSTNAME=<your local IP address> npm run serve
```

Comment:
The comment above "Start ng serve middleware binding to all interfaces" is incorrect. It start only on the interface given by `FE_HOST` or on `localhost` if not defined.

I wonder if that would be simpler to just use
```
FE_HOST=<your local IP address> npm run serve
```
without any `PROXY_HOSTNAME` and then update `frontend/package.json` to replace `--public-host http://${PROXY_HOSTNAME:-localhost}:${FE_PORT:-4200}"` with `--public-host http://${PROXY_HOSTNAME:-${FE_HOST:-localhost}}:${FE_PORT:-4200}"` so that it uses `PROXY_HOSTNAME` or `FE_HOST` or `localhost`, and then we can stop using `PROXY_HOSTNAME` while keeping it for legacy reasons.

Reply:
But that may be a bad idea as this documentation should work for everyone, and not everyone is developing using latest `dev`.
Meanwhile, we can still replace `--public-host http://${PROXY_HOSTNAME:-localhost}:${FE_PORT:-4200}"` with `--public-host http://${PROXY_HOSTNAME:-${FE_HOST:-localhost}}:${FE_PORT:-4200}"` and maybe at some point we can remove the `PROXY_HOSTNAME` from the command above..